Selection_Sort/Selection_Sort.rb

```diff
@@ -0,0 +1,28 @@
+# passing array in SelectionSort fuction 
+def SelectionSort(a)
```

Comment:
Use snake case for function name.
Also put a comment containing time and space complexity of the algorithm.

Reply:
I will change it as currently exam is going on